uber/utils.py

```diff
@@ -767,8 +767,10 @@ def _server_to_url(server):
         return 'https://{}/reggie'.format(host)
     elif path.startswith('uber'):
         return 'https://{}/uber'.format(host)
-    elif c.PATH == '/uber':
-        return 'https://{}{}'.format(host, c.PATH)
+    elif path == 'uber':
+        return 'https://{}/{}'.format(host, path)
+    elif path == 'rams':
```

Comment:
The reason this is hardcoded is because this function deals with OTHER servers -- if we compared the URL being called to the existing server, then we later removed `/rams/` from the URL of new servers, we suddenly wouldn't be able to make any calls to older servers from the new servers.